spec/filters/cidr_spec.rb

```diff
@@ -102,5 +104,52 @@
        insist { subject.get("tags") }.nil?
     end
   end
+  
+  describe "Load network list from a file" do
 
+    let(:network_path) {File.join(File.dirname(__FILE__), "..","files","network")}
+    let(:config) do
+      "filter { cidr { network_path => \"#{network_path}\" address => \"%{clientip}\" add_tag => \[\"matched\"] }}"
+    end
+
+    sample("clientip" => "192.168.1.1") do
+      insist { subject.get("tags") }.include?("matched")
+    end
+
+    sample("clientip" => "10.1.2.1") do
+      insist { subject.get("tags").nil? }
```

Comment:
Please do. I was writing my tests based on the ones that were already there. What would be the best option in this case? Where can I learn more about testing my plugin with rspec?